### DISCUSSION TITLE
Problem opening port for information scan

### DISCUSSION THREAD

**Discussion by overcride** [advplyr/audiobookshelf#936] [Q&A] [answered]
Hi,

I use Audiobookshelf with Docker on Synology.
I am using Synology reverse proxy with certificate. I added a rule to point my domain https://audiobookshelf.mydomain.fr. Websockets are enabled.
My problem is that information scanning does not work with my firewall enabled.
My port 80, 443 is open. I tested by opening port 13378 anyway but nothing...
@jrhedman seems to have solved the problem [ici](https://github.com/advplyr/audiobookshelf/issues/531#issuecomment-1207411709).

But I don't see which port to open. Do you have an idea ?

Thanks :)

**Comment by advplyr** [comment]
I'm not sure since I don't use Synology but you can also try posting about this in the Discord server.

**Comment by overcride** [comment, marked as answer]
Ok, I managed to solve my problem.

My firewall rules were connected to all my interfaces (LAN1, LAN2, etc...). I recreated the rules only on LAN1 and I no longer have a problem.

So there is no additional port to open.

Have a good day.
